README.md

```diff
@@ -496,6 +496,8 @@ docker compose build
 docker compose run client
 ```
 
+> Note that if you get an error like "'name' does not match any of the regexes: '^x-'" then you will need to upgrade your version of docker compose.
```

Comment:
Looked at the rendering of the README. In the section prior to this one, we include a note. Let's follow the same pattern as that one.

#### Pattern
- Make the statement italic
- Drop use of markdown quote (`>`)
- Replace `Note that` with `Note:`
- Highlight error message by wrapping in backticks

#### Result
_Note: If you get an error like `'name' does not match any of the regexes: '^x-'` then you will need to upgrade your version of docker compose._

Reply:
I made the change and pounded.